renderer/views/player.js

```diff
@@ -269,9 +269,9 @@ function renderCastScreen (state) {
     castIcon = 'tv'
     castType = 'DLNA'
     isCast = true
-  } else if (state.playing.location === 'vlc') {
+  } else if (state.playing.location === 'external') {
     castIcon = 'tv'
-    castType = 'VLC'
+    castType = 'External' // TODO: Should we use the player name instead? see `unsupported-media-modal.js`
```

Comment:
'External' is fine. If you want, you can extract the filename from the external player path and use that, defaulting to 'VLC' if no external player path is set.

This would show eg. 'QuickTime.app', 'mplayer', or 'VLC' instead of 'External'